File: App Web/Modelos/Historia.php
<?php
class Historia{
    private $id_historia;
    private $id_usuario;
    private $fecha_publicacion;
    private $imagen;
    private $likes;

    public static function obtenerTodas()
    {
        $conexion = Conexion::conectar(); // Tengo que pedir los datos que me hacen falta para cada publicacion en la API
        $sql = "SELECT 
                FROM historias
                JOIN usuarios ON id_usuario = id
                ORDER BY id_publicacion DESC";

        $stmt = $conexion->prepare($sql);
        $stmt->execute();

        return $stmt->fetchAll(PDO::FETCH_ASSOC);
    }
}

File: app/src/main/java/com/example/appmovil/Models/Historia.java
package com.example.appmovil.Models;

import java.io.Serializable;
import java.sql.Blob;
import java.sql.Date;

public class Historia implements Serializable {
    private int id_historia;
    private int id_usuario;
    private Date fecha_publicacion;
    private Blob imagen;
    private int likes;
}
